rust/xnvme-sys/Cargo.toml

```diff
@@ -1,10 +1,18 @@
 [package]
 name = "xnvme-sys"
-version = "0.0.0"
+version = "0.7.1"
```

Comment:
Is there a way to store the version in a variable to make sure we don't have a case where it is updated here but not in the bottom?
```
[package.metadata.system-deps]
xnvme = "0.7.1"
```

Reply:
I don't think so. You could use a template engine to generate the manifest maybe? Probably not worth it.

Reply:
Nevermind then :)